src/_pytest/main.py

```diff
@@ -349,12 +368,14 @@ def pytest_collection_modifyitems(items, config):
 
 
 class FSHookProxy:
-    def __init__(self, fspath, pm, remove_mods):
+    def __init__(
+        self, fspath: py.path.local, pm: PytestPluginManager, remove_mods
+    ) -> None:
         self.fspath = fspath
         self.pm = pm
         self.remove_mods = remove_mods
 
-    def __getattr__(self, name):
+    def __getattr__(self, name: str):
```

Comment:
Conflicts with https://github.com/pytest-dev/pytest/pull/6461.

Reply:
Taken into https://github.com/pytest-dev/pytest/pull/6461 (https://github.com/pytest-dev/pytest/pull/6461/commits/ae5d16be10c139da6a21eab34f7decbf93af721b).

Reply:
https://github.com/pytest-dev/pytest/pull/6461 was merged.
Should be rebased now after master has been merged into features.